src/app/Plans/components/VMStatusTable.tsx

```diff
@@ -36,7 +37,7 @@ const VMStatusTable: React.FunctionComponent<IVMStatusTableProps> = ({
               <Step status={status} index={index} />
             </FlexItem>
             <FlexItem>
-              <Text>{step.name}</Text>
+              <Text>{MigrationVMStepsType[step.name]}</Text>
```

Comment:
Re: handling the case where it is undefined, you could just use `MigrationVMStepsType[step.name] || step.name` to fall back on the name value itself

Reply:
Yes, that's what I just did.